### PR TITLE
Adding timeout value to the docs for toast messages

### PR DESCRIPTION
**Pull request by ejzn**

### CODE REVIEW

File: api/blackberry_ui_toast.js [hunk 1]
```diff
@@ -29,6 +29,7 @@ blackberry.ui.toast ={
          * @param {String} [options.buttonText] Optional parameter that specifies the label for the button within the toast.
          * @callback {function} [options.buttonCallback] Optional callback for receiving the button event when the user selects the toast button.
          * @callback {function} [options.dismissCallback] Optional callback for receiving an event when the toast dismisses either through the toast's timeout or when the button is selected.
+         * @param {Number} [options.timeout] Optional timeout value that can be used to extend the toast for as long as the user desires. The value specified is parsed in milliseconds.
```
Comment (jeffheifetz) [inline api/blackberry_ui_toast.js]: nitpick. The []'s are supposed to be used to denote that its optional, do we also need to do so in the description?

**Comment by ejzn** [comment]
@jeffheifetz Master needs to be updated I think. This pull request is showing quite diff.

**Comment by jeffheifetz** [comment]
@erikj54 Please rebase this branch off of blackberry-webworks:master. Device-locked status should not be merged into master (right @rwmtse ?)

**Comment by nukulb** [comment]
@erikj54 branch off master please.

**Pull request closed without merging**